### PR TITLE
Segres2D algo update

### PR DESCRIPTION
**Pull request by myron**
Refactored segresnet2d algo to be consistent with segresnet (3d)

### CODE REVIEW

File: auto3dseg/algorithm_templates/segresnet2d/scripts/algo.py [hunk 1]
```diff
@@ -48,7 +49,7 @@ def pre_check_skip_algo(self, skip_bundlegen: bool=False, skip_info: str=''):
                 skip_info = f'2D network is skipped due to median spacing of {spacing}.'
```
Comment (mingxin-zheng) [inline auto3dseg/algorithm_templates/segresnet2d/scripts/algo.py:49]: Can we improve the info check:
```
f'2D network is skipped due to median spacing of {spacing}.'
```
by adding:
```
f"SegresNet2D is skipped due to median spacing of {spacing}, which means the dataset is not highly anisotropic, e.g.(res_z < 3*(res_x + rex_y)/2)."
```

Comment (myron) [reply auto3dseg/algorithm_templates/segresnet2d/scripts/algo.py]: ok, I updated that

**Review by mingxin-zheng — APPROVED** [review]

**Pull request merged**